src/main/java/com/dell/isg/smi/service/server/configuration/manager/ConfigurationManagerImpl.java

```diff
@@ -307,51 +314,37 @@ private SHARE_TYPES getShareTypeEnum(int type) {
 	/**
 	 * This Method updates the XML Configuration File for a given Component
 	 * Lists to be updated
-	 * 
+	 *
 	 * @param request-
 	 *            The ComponentList request
 	 * @return the list of updated ServerComponents
 	 * @throws Exception
 	 */
 	private List<ServerComponent> updateConfigurationFile(ComponentList request) throws Exception {
 		List<ServerComponent> updatedServerComponents = null;
-		try {
-			if (null != request) {
+		try
+		{
+			if (null != request)
+			{
 				DocumentBuilderFactory dbf = DocumentBuilderFactory.newInstance();
 				DocumentBuilder db = dbf.newDocumentBuilder();
 				String filePathName = request.getServerAndNetworkShareRequest().getFilePathName();
 				logger.info("updateConfigurationFile: File Name from updated request:: " + filePathName);
 				File xml = new File(filePathName);
 				Document document = db.parse(xml);
-				JAXBContext jaxbContext = JAXBContextFactory.createContext(new Class[] { SystemConfiguration.class },
-						null);
-
-				Binder<Node> binder = jaxbContext.createBinder();
-				binder.setProperty(Marshaller.JAXB_FORMATTED_OUTPUT, true);
-				binder.setProperty(Marshaller.JAXB_FRAGMENT, true);
-
-				Node xmlNode = document.getDocumentElement();
-
-				SystemConfiguration systemConfig = (SystemConfiguration) binder.unmarshal(document);
-
-				updatedServerComponents = updateXMLSystemConfiguration(systemConfig, request);
-
```

Comment:
Also you can delete `ComponentPredicate.updateXMLSystemConfiguration` and `ComponentPredicate.filterRequestServerComponents`